src/main.go

```diff
@@ -293,10 +308,8 @@ func main() {
 		log.Infof("Alive %d goroutines after cleaning up.\n", runtime.NumGoroutine())
 		os.Exit(0)
 	}()
-	log.Infoln(http.ListenAndServe(port, cors.Default().Handler(router)).Error())
-	// keep the below line to the end of file so that we ensure that we give a confirmation message only when all the
-	// required resources for the application is up and healthy.
 	log.Infoln("Bench-routes is up and running")
+	log.Infoln(http.ListenAndServe(port, cors.Default().Handler(router)).Error())
```

Comment:
Code is always blocking on this line...

Reply:
It is meant to be blocked here. Do we not want to block it?

Reply:
Of course we would. But printing "bench routes is up and running" after this line would never be called so I moved it up